src/software/ai/hl/stp/tactic/assigned_skill/assigned_skill_tactics.h

```diff
@@ -1,14 +1,24 @@
 #pragma once
 
 #include "software/ai/hl/stp/tactic/assigned_skill/assigned_skill_tactic.hpp"
+#include "software/ai/hl/stp/skill/chip/chip_skill_fsm.h"
 #include "software/ai/hl/stp/skill/kick/kick_skill_fsm.h"
 #include "software/ai/hl/stp/skill/shoot/shoot_skill_fsm.h"
 
+class ChipSkillTactic : public AssignedSkillTactic<ChipSkillFSM, GetBehindBallSkillFSM>
+{
+    using AssignedSkillTactic::AssignedSkillTactic;
+};
```

Comment:
does `COPY_TACTIC(ChipSkillTactic, AssignedSkillTactic<ChipSkillFSM, GetBehindBallSkillFSM>)` not work?

Reply:
No bc can't have angle brackets and commas in a macro argument